src/adapters/aol.js

```diff
@@ -2,17 +2,130 @@ const utils = require('../utils.js');
+  const nexageBaseApiTemplate = template`${'protocol'}://${'host'}/bidRequest?`;
+  const nexageGetApiTemplate = template`dcn=${'dcn'}&pos=${'pos'}&cmd=bid${'ext'}`;
   const BIDDER_CODE = 'aol';
-  const SERVER_MAP = {
+  const MP_SERVER_MAP = {
     us: 'adserver-us.adtech.advertising.com',
     eu: 'adserver-eu.adtech.advertising.com',
     as: 'adserver-as.adtech.advertising.com'
   };
+  const NEXAGE_SERVER = 'hb.nexage.com';
+  const SYNC_TYPES = {
+    iframe: 'IFRAME',
+    img: 'IMG'
+  };
+
+  let domReady = (() => {
+    let readyEventFired = false;
+    return fn => {
+      let idempotentFn = () => {
+        if (readyEventFired) {
+          return;
+        }
+        readyEventFired = true;
+        return fn();
+      };
+      let doScrollCheck = () => {
+        if (readyEventFired) {
+          return;
+        }
+        try {
+          document.documentElement.doScroll('left');
+        } catch (e) {
+          setTimeout(doScrollCheck, 1);
+          return;
+        }
+        return idempotentFn();
+      };
+      if (document.readyState === "complete") {
+        return idempotentFn();
+      }
+      if (document.addEventListener) {
+        document.addEventListener("DOMContentLoaded", idempotentFn, false);
+        window.addEventListener("load", idempotentFn, false);
+      } else if (document.attachEvent) {
+        document.attachEvent("onreadystatechange", idempotentFn);
+        window.attachEvent("onload", idempotentFn);
+        let topLevel = false;
+        try {
+          topLevel = window.frameElement === null;
+        } catch (e) {
+        }
+        if (document.documentElement.doScroll && topLevel) {
+          return doScrollCheck();
+        }
+      }
+    };
+  })();
+
+  function dropSyncCookies(pixels) {
+    let pixelElements = parsePixelItems(pixels);
+    renderPixelElements(pixelElements);
+  }
+
+  function parsePixelItems(pixels) {
+    let itemsRegExp = /(img|iframe)[\s\S]*?src\s*=\s*("|')(.*?)\2/gi;
+    let tagNameRegExp = /\w*(?=\s)/;
+    let srcRegExp = /src=("|')(.*?)\1/;
+    let pixelsItems = [];
+
+    if (pixels) {
+      let matchedItems = pixels.match(itemsRegExp);
+      if (matchedItems) {
+        matchedItems.forEach(item => {
+          let tagNameMatches = item.match(tagNameRegExp);
+          let sourcesPathMatches = item.match(srcRegExp);
+          if (tagNameMatches && sourcesPathMatches) {
+            pixelsItems.push({
+              tagName: tagNameMatches[0].toUpperCase(),
+              src: sourcesPathMatches[2]
+            });
+          }
+        });
+      }
+    }
+
+    return pixelsItems;
+  }
+
+  function renderPixelElements(pixelsElements) {
+    pixelsElements.forEach((element) => {
+      switch (element.tagName) {
+        case SYNC_TYPES.img:
+          return renderPixelImage(element);
```

Comment:
Check if you can reuse these functions 
https://github.com/prebid/Prebid.js/blob/master/src/utils.js#L446